### PR TITLE
fixed a typo and replaced placeholderkv with valkey

### PR DESCRIPTION
**Pull request by daniel-house**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,24 +8,24 @@ The path follows the pattern: ``clients/{language}/github.com/{owner}/{repositor
...
 ## Commands
 
-PlaceHolderKV commands are described in the `commands.json` file that is auto generated
-from the PlaceHolderKV repo based on the JSON files in the commands folder.
-See: https://github.com/placeholderkv/placeholderkv/tree/unstable/src/commands
-See: https://github.com/placeholderkv/placeholderkv/placeholderkv/unstable/utils/generate-commands-json.py
+Valkey commands are described in the `commands.json` file that is auto generated
+from the Valkey repo based on the JSON files in the commands folder.
+See: https://github.com/valkey/valkey/tree/unstable/src/commands
+See: https://github.com/valkey/valkey/valkey/unstable/utils/generate-commands-json.py
```
Comment (zuiderkwast) [inline README.md:28]:
```suggestion
See: https://github.com/valkey-io/valkey/tree/unstable/src/commands
See: https://github.com/valkey-io/valkey/valkey/unstable/utils/generate-commands-json.py
```

File: README.md [hunk 2]
```diff
@@ -8,24 +8,24 @@ The path follows the pattern: ``clients/{language}/github.com/{owner}/{repositor
 The ``{language}`` component of the path is the path-safe representation
 of the full language name which is mapped in [languages.json](./languages.json).
 
-Each client's JSON object represents the details displayed on the [clients documentation page](https://placeholderkv.io/docs/clients).
+Each client's JSON object represents the details displayed on the [clients documentation page](https://valkey.io/docs/clients).
 
-For example [clients/python/github.com/placeholderkv](./clients/python/github.com/placeholderkv/placeholderkv-py.json):
+For example [clients/python/github.com/valkey](./clients/python/github.com/valkey/valkey-py.json):
```
Comment (zuiderkwast) [inline README.md:13]: There is no client with that name. Can you find another example which actually exists? Look at redis.io website and the files under clients/ for inspiration.

File: README.md (hunk 1 above)
Comment (PingXie) [reply README.md:28]:
```suggestion
See: https://github.com/valkey/valkey/tree/unstable/src/commands
See: https://github.com/valkey/valkey/tree/unstable/utils/generate-commands-json.py
```

File: README.md (hunk 2 above)
Comment (daniel-house) [reply README.md:13]: Both of those are great ideas, but outside the scope of this PR.

**Comment by daniel-house** [comment]
Because the new commit fails the DCO check, this PR cannot be merged.

**Comment by zuiderkwast** [comment]
You can edit the commit messages using interactive rebase and then force-push.

**Comment by daniel-house** [comment]
That would require that I sign-off for PingXie, which would be inappropriate.

**Comment by zuiderkwast** [comment]
I believe it's OK to sign off on other people's code, at least short things like this. See https://github.com/valkey-io/valkey/blob/unstable/CONTRIBUTING.md#developer-certificate-of-origin "(b)"

**Comment by daniel-house** [comment]
Sorry, no, I will not sign off on someone else's commit.  He can fix it himself.

**Comment by zuiderkwast** [comment]
Of course, I mean it is allowed (on your own responsibility) but it's not required. :)

**Pull request closed without merging**